.circleci/config.yml

```diff
@@ -128,7 +129,21 @@ jobs:
           password: $DOCKERHUB_PASSWORD
     steps:
       - checkout
-      - run: bash scripts/detect-package-json-changes.sh
+      - install_container_dependencies
+      - run:
```

Comment:
I think further down, around line 245/260, you can make licenses_test require lint_unit_test_coverage. Then it's already been checked out for you and you don't have to do npm ci.

Or maybe it's easier to just get rid of licenses_test as a job, and just move lines 141 /146 into lint_unit_test_coverage.

Sort of nit-picky, but it would be nice to avoid the duplication of running npm ci, both in terms of code duplication and it executing twice.

Reply:
I'll try requiring lint_unit_test_coverage. I tried just putting it in the lint_unit_test_coverage job, but I got errors that the files were different. I couldn't reproduce them being different locally. I'm not sure what step in lint_unit_test_coverage makes that happen.

Reply:
Seems like you have to do the checkout, but got rid of the npm ci